### DISCUSSION TITLE
bug fix

### DISCUSSION THREAD

**Discussion by tybreizh29** [oamg/convert2rhel#874] [Show and tell]
Hi
I did make some changes to the sources to try to make it work to convert my offline rocky 9.2 to redhat 9.2.
I found a bug in one your file.
if package name is "A.Y.arch" you raise an exception because you are expecting only one "."
file : /usr/lib/python3.9/site-packages/convert2rhel/pkgmanager/handlers/dnf/__init__.py
line 124 : name, arch = tuple(pkg.rsplit(".",1))          <- I added ",1" to extract only the arch 
best regards
m
arc

**Comment by Venefilyn** [comment]
There is a PR opened to convert el9 systems at the moment it isn't possible 

However, without seeing the logs this looks like a bug we fixed in 1.3.2

**Comment by bocekm** [comment]
As Eric wrote, this has been fixed in convert2rhel 1.3.2. This is the fix specifically: https://github.com/oamg/convert2rhel/pull/855.
